README.md

```diff
@@ -1,2 +1,788 @@
 # simplr-flux
-Flux architecture.
+
+Flux implementation that enables you to handle actions by their type, without nasty and never ending if or switch trees.
+
+The package is most useful when used with [TypeScript](https://www.typescriptlang.org/) and [React](https://www.typescriptlang.org/).
+
+## Get started
+
+```sh
+npm install simplr-flux --save
+```
+
+## Concepts
+
+Basic concepts of `SimplrFlux` are no different from [original flux](https://facebook.github.io/flux/) [concepts](https://github.com/facebook/flux/tree/master/examples/flux-concepts).
+
+We also recommend you to follow [best practices](https://facebook.github.io/flux/docs/flux-utils.html#best-practices) original flux is proposing.
```

Comment:
We also recommend you to follow [best practices](https://facebook.github.io/flux/docs/flux-utils.html#best-practices) original flux is proposing.
->
We also recommend to follow [the best practises](https://facebook.github.io/flux/docs/flux-utils.html#best-practices) proposed by original flux.